docs/source/features/communication_overlap.rst

```diff
@@ -0,0 +1,61 @@
+Communication Overlap
+====================
+
+Data-parallel communication overlap
+-----------------------------------
+
+NeMo supports the overlap of the data-parallel (DP) communications with the computations in LLM training.
```

Comment:
Should we say DDP or DP?

Reply:
It should be DP. DP is a parallelism method and DDP is an implementation.